### DISCUSSION TITLE
Search User feature

### DISCUSSION THREAD

**Discussion by socialmediaboosterlb** [davidteather/TikTok-Api#547] [Q&A] [answered]
i noticed the search user feature was removed, will it ever be reimplemented? and why was it removed in the first place

**Comment by davidteather** [comment, marked as answer]
This feature was not removed. Although admittedly I didn't realize that there was an error in the code see #545 

Example usage
```
api.search_for_users('target')
```

**Reply by socialmediaboosterlb** [reply]
i just tried it, still not working. is there any expectations when will this feature be working again? thanks a lot for your effort in the upkeep of this amazing api btw

**Reply by davidteather** [reply]
The method is working, your code is the problem here. I recommend watching the youtube video linked in the readme on how to setup the package. Also, remember to update your package with `pip install TikTokApi --upgrade`
